plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/sql/SQLDialect.java

```diff
@@ -441,7 +441,11 @@ default boolean supportsAliasInConditions() {
      */
     String formatStoredProcedureCall(DBPDataSource dataSource, String sqlText);
 
-    void generateStoredProcedureCall(StringBuilder sql, DBSProcedure proc, Collection<? extends DBSProcedureParameter> parameters);
+    /**
+     * Generates stored procedure call. Parameters (optionally) can be surrounded by cast(:param as paramType).
+     */
+    void generateStoredProcedureCall(StringBuilder sql, DBSProcedure proc, Collection<? extends DBSProcedureParameter> parameters,
+                                     boolean castParams);
```

Comment:
Please specify nullability annotations for the method's parameters. Don't forget about its implementations.

```suggestion
    void generateStoredProcedureCall(
        StringBuilder sql,
        DBSProcedure proc,
        Collection<? extends DBSProcedureParameter> parameters,
        boolean castParams);
```